src/Admin/SiteHealth.php

```diff
@@ -264,6 +275,17 @@ public function css_transient_caching() {
 			'status'      => $status,
 			'label'       => esc_html( $label ),
 		];
+
+		if ( $disabled ) {
+			$data['actions'] = sprintf(
+				'<p><a class="button reenable-css-transient-caching" href="#">%s</a><span class="dashicons dashicons-yes success-icon"></span><span class="dashicons dashicons-no failure-icon"></span><span class="success-text">%s</span><span class="failure-text">%s</span></p>',
+				__( 'Re-enable transient caching', 'amp' ),
+				__( 'Reload the page to refresh the diagnostic check.', 'amp' ),
+				__( 'The operation failed, please reload the page and try again.', 'amp' )
```

Comment:
Would these warrant `esc_html__()`? I know the others aren't doing it.

Reply:
Yes, we should escape them.